### ISSUE TITLE
Error in types included with 0.4.1

### ISSUE THREAD

**Issue by dasa** [GoogleChromeLabs/squoosh#1219]
**Before you start**
Please take a look at the [FAQ](https://github.com/GoogleChromeLabs/squoosh/wiki/FAQ) as well as the already opened issues! If nothing fits your problem, go ahead and fill out the following template:

**Describe the bug**

`tsc` errors:

```
node_modules/@squoosh/lib/build/codecs.d.ts:1:32 - error TS2503: Cannot find namespace 'EmscriptenWasm'.

1 interface DecodeModule extends EmscriptenWasm.Module {
                                 ~~~~~~~~~~~~~~

node_modules/@squoosh/lib/build/codecs.d.ts:22:59 - error TS2307: Cannot find module '../../codecs/mozjpeg/enc/mozjpeg_enc' or its corresponding type declarations.

22 import type { MozJPEGModule as MozJPEGEncodeModule } from '../../codecs/mozjpeg/enc/mozjpeg_enc';
                                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@squoosh/lib/build/codecs.d.ts:23:60 - error TS2307: Cannot find module '../../codecs/mozjpeg/enc/mozjpeg_enc' or its corresponding type declarations.

23 import type { EncodeOptions as MozJPEGEncodeOptions } from '../../codecs/mozjpeg/enc/mozjpeg_enc';
                                                              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@squoosh/lib/build/codecs.d.ts:24:53 - error TS2307: Cannot find module '../../codecs/webp/enc/webp_enc' or its corresponding type declarations.

24 import type { WebPModule as WebPEncodeModule } from '../../codecs/webp/enc/webp_enc';
                                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@squoosh/lib/build/codecs.d.ts:25:57 - error TS2307: Cannot find module '../../codecs/webp/enc/webp_enc.js' or its corresponding type declarations.

25 import type { EncodeOptions as WebPEncodeOptions } from '../../codecs/webp/enc/webp_enc.js';
                                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@squoosh/lib/build/codecs.d.ts:26:53 - error TS2307: Cannot find module '../../codecs/avif/enc/avif_enc' or its corresponding type declarations.

26 import type { AVIFModule as AVIFEncodeModule } from '../../codecs/avif/enc/avif_enc';
                                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@squoosh/lib/build/codecs.d.ts:27:57 - error TS2307: Cannot find module '../../codecs/avif/enc/avif_enc.js' or its corresponding type declarations.

27 import type { EncodeOptions as AvifEncodeOptions } from '../../codecs/avif/enc/avif_enc.js';
                                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@squoosh/lib/build/codecs.d.ts:28:51 - error TS2307: Cannot find module '../../codecs/jxl/enc/jxl_enc' or its corresponding type declarations.

28 import type { JXLModule as JXLEncodeModule } from '../../codecs/jxl/enc/jxl_enc';
                                                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@squoosh/lib/build/codecs.d.ts:29:56 - error TS2307: Cannot find module '../../codecs/jxl/enc/jxl_enc.js' or its corresponding type declarations.

29 import type { EncodeOptions as JxlEncodeOptions } from '../../codecs/jxl/enc/jxl_enc.js';
                                                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@squoosh/lib/build/codecs.d.ts:30:51 - error TS2307: Cannot find module '../../codecs/wp2/enc/wp2_enc' or its corresponding type declarations.

30 import type { WP2Module as WP2EncodeModule } from '../../codecs/wp2/enc/wp2_enc';
                                                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@squoosh/lib/build/codecs.d.ts:31:56 - error TS2307: Cannot find module '../../codecs/wp2/enc/wp2_enc.js' or its corresponding type declarations.

31 import type { EncodeOptions as WP2EncodeOptions } from '../../codecs/wp2/enc/wp2_enc.js';
                                                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@squoosh/lib/build/codecs.d.ts:32:28 - error TS2307: Cannot find module '../../codecs/png/pkg/squoosh_png.js' or its corresponding type declarations.

32 import * as pngEncDec from '../../codecs/png/pkg/squoosh_png.js';
                              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**To Reproduce**

Use package in a TS project.

**Expected behavior**

TypeScript does not error.

**Version:**
 - TypeScript version: 4.5.5

**Comment by jakearchibald** [comment]
Please provide a way to reliably reproduce this error.

For example, I'm not seeing the error [here](https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAbzgSRAQwOYFMAKEIA2cAvnAGZQQhwDkAAgM4COArvgwBYD0BwARjQDcAKFCRY5BuUrUaZBlzAzgDLAyHDhAYwgA7BvFCZc+IgF44urAHcU6bHkIAKACwBKETv2H7WOBaMHUwA6YF1sA1RjJzRrNGB4eWCoLDQAEwAxYAIsJxpgxTQYbhgILkCsYLBwmjcPTS8DOCUsJQgtNQYAeTAYYD0pCwRhODguLgB1DixdOD4IYrhrYDTFtF00uGngDA54NBS4BjAsLWAyYCw0gBo4Yr8KuBSGYAAvK7uII5Ozi4+X97BEZPNRvLAALkQwNGy1WHEhAEYAAxI67QrZYHZ7SEAVlRwOIaNGXAAVMDxlMZnBMfcoEsVotoBisYYpMdTudLjc7tM4I9nmDNqVvhy-psAX4ANZYVphDBwNDsrTwKBFfpA0YC96Q4ajGEM+FwZH40bEYEkrjCYgiWLxHzGKopNodBgMJwtZ2dHp9Ab1bQDeAzHRpLDe-r6fxQ0YgCCvABWJwwOsJYy462p4BgAE84BA+HHTvAQKkIzQWKo4CGyGgWAR4KoYH1wupgXGAB4EHXo1hoXjZyEAThNJDR1v93hBDFr8AstoSfN8wSDEBDTmXIbDvsEQA).

I'll reopen this when a reduced reproduction is provided.

**Comment by jakearchibald** [comment]
Someone else provided a reduced test. This will be fixed in 0.5.0. I've deprecated 0.4.1.
